backend/backup_local.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+BUCKET=gsa-fac-private-s3
+ENVIRONMENT=LOCAL
+#echo "Environment set as: $1"
+#export PATH=/home/vcap/deps/0/apt/usr/lib/postgresql/15/bin:$PATH
+DATE=$(date '+%Y-%m-%d-%H%M')
+LOG_FILE="$DATE"_backup_log.txt 
+DB_BACKUP_FILE="$DATE"-"$ENVIRONMENT"-db-backup.psql.bin
+MEDIA_BACKUP_FILE="$DATE"-"$ENVIRONMENT"-media-backup.tar
```

Comment:
```suggestion
DB_BACKUP_FILE="$DATE"-"$ENVIRONMENT"-db-backup.psql.bin
MEDIA_BACKUP_FILE="$DATE"-"$ENVIRONMENT"-media-backup.tar
```
These should be removed, as they will be passed in from GHA

Reply:
This file is only for testing locally in a docker env. I think we'll remove it from our codebase once we've made similar changes to backup_database.sh and tested it in the `dev` space.